cypress/support/commands.js

```diff
@@ -0,0 +1,184 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add("login", (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add("drag", { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add("dismiss", { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This is will overwrite an existing command --
+// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+const stringify = JSON.stringify;
+const backendUrl = Cypress.config('backendUrl');
+const serverRestartDelay = Cypress.config('serverRestartDelay');
+
+Cypress.Commands.add('createUser', () => {
+  const user = {
+    username: 'soup',
+    email: 'hi@strapi.io',
+    password: 'coucou123',
+  };
+
+  return cy.request({ url: `${backendUrl}/users-permissions/init`, method: 'GET' })
+    .then(response => {
+      const { body: { hasAdmin } } = response;
+
+      if (!hasAdmin) {
+        // Create one
+        cy.request({ url: `${backendUrl}/auth/local/register`, method: 'POST', body: user });
+      }
+    });
+});
+
+Cypress.Commands.add('checkModalOpening', () => {
+  return cy.get('.modal').invoke('show');
+})
+
+Cypress.Commands.add('deleteUser', (id, jwt) => {
+  cy.request({
+    url: `${backendUrl}/users/${id}`,
+    method: 'DELETE',
+    headers: {
+      Authorization: `Bearer ${jwt}`
```

Comment:
Are you sure there is no way to only pass the Bearer token only once? Then, all the future requests will be automatically authenticated?

Reply:
I'm creating new token every new test